tests/cpp_validation_tests/test_validation.cpp

```diff
@@ -608,12 +608,15 @@ TEST_CASE("Validation test single") {
 TEST_CASE("Validation test batch") {
     std::vector<CaseParam> const& all_cases = get_all_batch_cases();
     for (CaseParam const& param : all_cases) {
-        SUBCASE(param.case_name.c_str()) {
-            try {
-                validate_batch_case(param);
-            } catch (std::exception& e) {
-                auto const msg = std::string("Unexpected exception with message: ") + e.what();
-                FAIL_CHECK(msg);
+        if (param.case_name ==
```

Comment:
Why do we have this for a specific validation case?

Reply:
test logic. to be disabled